index.js

```diff
@@ -3,8 +3,13 @@
  * @typedef {import('hast').Properties} Properties
  * @typedef {import('hast').Element['children'][number]} ElementChild
  *
+ * @typedef {Options | ((node: import('hast').Element, tree: Root) => Options)} inputOptions
+ *   Configuration as either a function or an object.
+ *
```

Comment:
```suggestion
```

This can’t work in several cases with unified because plugins are functions. And functions are plugins.

Reply:
Ok, I'll make each individual options config property a function